src/compiler/types.ts

```diff
@@ -741,6 +742,13 @@ namespace ts {
         expression: Expression;
     }
 
+    export interface PrivateName extends Declaration {

```

Comment:
Should extend `Node` not `Declaration`

Reply:
thanks @RyanCavanaugh ! I'll check this out.

Reply:
Ah, I was mistaken on this one. `extends Declaration` is fine